res/values/resurrection_device_maintainers_strings.xml

```diff
@@ -592,6 +592,9 @@
   <string name="device_land">Xiaomi Redmi 3s</string>
   <string name="device_land_summary">land\nMaintainer - Saicharan </string>
 
+  <string name="device_santoni">Xiaomi Redmi 4X</string>
+  <string name="device_santoni_summary">santoni\nMaintainer - IsaacChen </string>
```

Comment:
Oh Thanks a lot for instant reply. Santoni's ROM dev is scattered at common Android development, Is https://forum.xda-developers.com/showpost.php?p=73396004&postcount=22180 enough or how to get Santoni subforum in XDA?..